docs/articles/sdk/configure-qtcreator.rst

```diff
@@ -0,0 +1,52 @@
+Configuring QtCreator to use the SDK
+====================================
+
+After the installation of the SDK and sourcing environment, there are couple of
+steps to do for running QtCreator in target environment. This roadmap assumes
+that SDK and QtCreator should be installed and environments are sourced already.
+
+Adding a new kit
+--------------------
+
+To be able to cross-compile via QtCreator, new build kit should be added to
+QtCreator.
+
+* Open QtCreator
+* Open Options form from Tools->Options, then click ``Build & Run`` item on the
+left panel of option window.
+* Select ``Kits`` tab and click ``Add`` button.
+* Name your new kit and choose ``Device type`` as ``GenericLinuxOsType``.
+* Then browse and submit the sysroot. If you don't know sysroot location you can
+discover by typing ``echo ${SDKTARGETSYSROOT}`` to console.
+* Then choose C compiler. If cross compiler of the SDK is not listed, it should
+be added as a new gcc. To add new gcc click ``Manage`` button next to compiler.
+Add you GCC compiler path here. You can discover this path from environment as well.
+Just type ``which ${CC}`` to console to discover.
+* Choose C++ compiler as you did in previous step. You can discover path by typing
+``which ${CXX}`` to console.
+* Copy the environment list of the SDK to ``Environment``. You can discover environment
+list by typing ``env`` to console.
+* Choose ``Debugger``, if it is not listed in dropdown menu click ``Manage..``
+and add a new Debugger. You can discover debugger path with command ``which $GDB``
+* Choose CMake Tool, if it is not listed in dropdown menu click ``Manage..``
+button and add a new CMake. You can discover Cmake path with command : ``which cmake``
+* If you are working with Qmake then it should be added as well. You can discover
+qmake path with command ``which qmake``
+* Add ``CMAKE_SYSROOT:STRING=<path of sysroot>`` item to ``CMake Configuration``
+list. You can discover sysroot path by typing ``echo ${SDKTARGETSYSROOT}`` to console.
+
+
+How to open the template service project with QtCreator
+-------------------------------------------------------
+
+* Open QtCreator
+* From ``File` menu choose ``New File or Project``
```

Comment:
"From the ``File`` menu choose ..."

You missed the "the" and the closing ` with File.